[2023-10-01T12:31:50.582+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Valdi_Pipeline.get_data_from_postgresql manual__2023-10-01T11:31:45.520154+00:00 [queued]>
[2023-10-01T12:31:50.595+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Valdi_Pipeline.get_data_from_postgresql manual__2023-10-01T11:31:45.520154+00:00 [queued]>
[2023-10-01T12:31:50.596+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2023-10-01T12:31:50.597+0000] {taskinstance.py:1369} INFO - Starting attempt 2 of 2
[2023-10-01T12:31:50.597+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2023-10-01T12:31:50.614+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): get_data_from_postgresql> on 2023-10-01 11:31:45.520154+00:00
[2023-10-01T12:31:50.630+0000] {standard_task_runner.py:52} INFO - Started process 937 to run task
[2023-10-01T12:31:50.634+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Valdi_Pipeline', 'get_data_from_postgresql', 'manual__2023-10-01T11:31:45.520154+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_Rivaldi_Valensia_DAG.py', '--cfg-path', '/tmp/tmp33z3mz75', '--error-file', '/tmp/tmpg159zdf9']
[2023-10-01T12:31:50.636+0000] {standard_task_runner.py:80} INFO - Job 9: Subtask get_data_from_postgresql
[2023-10-01T12:31:50.728+0000] {task_command.py:371} INFO - Running <TaskInstance: Valdi_Pipeline.get_data_from_postgresql manual__2023-10-01T11:31:45.520154+00:00 [running]> on host aede7beb5ed8
[2023-10-01T12:31:50.850+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Valdi
AIRFLOW_CTX_DAG_ID=Valdi_Pipeline
AIRFLOW_CTX_TASK_ID=get_data_from_postgresql
AIRFLOW_CTX_EXECUTION_DATE=2023-10-01T11:31:45.520154+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-10-01T11:31:45.520154+00:00
[2023-10-01T12:31:51.900+0000] {python.py:173} INFO - Done. Returned value was: None
[2023-10-01T12:31:51.923+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=Valdi_Pipeline, task_id=get_data_from_postgresql, execution_date=20231001T113145, start_date=20231001T123150, end_date=20231001T123151
[2023-10-01T12:31:51.989+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-01T12:31:52.090+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
